### DISCUSSION TITLE
Improve scannability of sections table

### DISCUSSION THREAD

**Discussion by rensverschuren** [craftcms/cms#8439] [Ideas]
At `/admin/settings/sections` in the control panel, it is difficult to scan the table. For example, when I want to edit the entry types for a specific section (click on `Edit entry type`), the section name is at the far left of the table and the `Edit entry type` link is located at the far right. It is is difficult to see which link belongs to which section (especially on larger screens).

Some suggestions:

1. Move the entry types column more to the left;
2. Add a hover state to the row;
3. Maybe add the entry types settings to the section settings itself, maybe with a tab or something like that. (?)

**Comment by brandonkelly** [comment]
Agree it could be improved. Adding a row hover state probably makes sense for all "admin" tables @benjamindavid
